backend/src/candidates.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express')
+const candidates = express.Router()
+const { errorWrap } = require('./middleware')
+const { Candidate } = require('./models')
+
+candidates.get(
+  '/',
+  errorWrap(async (req, res) => {
+    let candidates
+    if (req.query.status) {
+      if (req.query.status == 'everyone') {
+        candidates = await Candidate.find()
+      } else {
+        candidates = await Candidate.find({ status: req.query.status })
+      }
+    } else {
+      candidates = await Candidate.find()
+    }
+    res.json({ result: candidates })
+  })
+)
+
+candidates.post(
+  '/',
+  errorWrap(async (req, res) => {
+    const c = new Candidate({
+      name: 'Tim',
+      email: 'other@gmail.com',
+      graduationDate: '2018',
+      major: 'CompE',
+      resumeID: 'resume2.pdf',
+      role: 'SWE'
+    })
+    await c.save()
+  })
+)
+
+candidates.get(
+  '/:candidateId',
+  errorWrap(async (req, res) => {
+    const candidate = await Candidate.findById(req.params.candidateId)
+    res.json({ result: candidate })
+  })
+)
+
+module.exports = candidates

```

Comment:
even if you do `module.exports = router` you can still do a `const candidates = require('./candidates')`. the names don't have to match up!